// containers/furnitureItem.js
import React, { Component } from 'react';
import { connect } from 'react-redux';
import { 
	fetchFurnitureById,
	addFurnitureToBasket 
} from 'actions';
import { getFurnitureById } from 'selectors';
import {Link} from 'react-router';


class FurnitureItem extends Component {
	componentDidMount() {
		this.props.fetchFurnitureById(this.props.params.id)
	}
	renderImage() {
		const {furnitureItem} = this.props;
		return (
			<img 
				className="furnitureItem"
				src={furnitureItem.image}
				alt={furnitureItem.name}
				width="400"
				height="550"
			/>
	)}

	renderContent() {
		const {furnitureItem} = this.props;
		const {addFurnitureToBasket} = this.props;
		return (
			<div>
				<ul className='contentList'>
				<li>Category: {furnitureItem.categoryName}</li>
				<li>Description: 
					<br />{furnitureItem.description}
				</li>
				<li><b>Price:</b> {furnitureItem.price}$</li>
			</ul>
			<Link 
				to='/furniture'
				className='btn btn-default btn-lg btn-item pull-left'
			>
				Back
			</Link>
			<button 
				className='btn btn-primary btn-lg btn-item pull-right'
				onClick={() => addFurnitureToBasket(furnitureItem.id)}
			>
				Buy now!
			</button>
			</div>
			
	)}
	render() {
		const {furnitureItem} = this.props;
		return (
			<div className='container'>
				<div className='row'>
				{furnitureItem && <h2 className="furnitureItem">{furnitureItem.name}</h2>}
					<div className='col-md-6'>
						{furnitureItem && this.renderImage()}
					</div>
					<div className='col-md-6'>
						{furnitureItem && this.renderContent()}
					</div>
				</div>	
			</div>
		)
	}
}
// console.log(getState())
const mapStateToProps = state => ({
	furnitureItem: getFurnitureById(state, state.furniturePages.id)
})

const mapDispatchToProps = {
	fetchFurnitureById,
	addFurnitureToBasket
}

export default connect(mapStateToProps, mapDispatchToProps)(FurnitureItem);
