// api/mockFurniture.js
export default [
	{
		id: "1",
		categoryId: "1",
		name: "DARK SEAT",
		categoryName: "CHAIRS",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. ",
    price: 65,
		image: "/uploads/dark_chair.jpg"
	},
	{
		id: "2",
		categoryId: "1",
		name: "RED SEAT",
		categoryName: "CHAIRS",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. ",
    price: 72,
		image: "/uploads/red_chair.jpg"
	},
	{
		id: "3",
		categoryId: "1",
		name: "BLUE SEAT",
		categoryName: "CHAIRS",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. ",
    price: 74,
		image: "/uploads/blue_chair.jpg"
	},
	{
		id: "4",
		categoryId: "1",
		name: "WHITE SEAT",
		categoryName: "CHAIRS",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. ",
    price: 99,
		image: "/uploads/white_chair.jpg"
	},
	{
		id: "5",
		categoryId: "2",
		name: "BlACK MODERN",
		categoryName: "BEDS",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. ",
    price: 165,
		image: "/uploads/black_bad.jpg"
	},
	{
		id: "6",
		categoryId: "3",
		name: "WHITE SEAT",
		categoryName: "TABLES",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. ",
    price: 134,
		image: "/uploads/white_table.jpg"
	},
	{
		id: "7",
		categoryId: "2",
		name: "FOR CHILDREN",
		categoryName: "BEDS",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. ",
    price: 104,
		image: "/uploads/childre_bed.jpg"
	},
	{
		id: "8",
		categoryId: "3",
		name: "ANTIQUARE",
		categoryName: "TABLES",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. ",
    price: 66,
		image: "/uploads/antiq_table.jpg"
	}
    
]
